documentcloud/entities/views.py

```diff
@@ -0,0 +1,40 @@
+from documentcloud.entities.permissions import IsOwnerOrReadOnly
+from .models import Entity
+from rest_framework import viewsets
+from rest_framework import permissions
+from .serializers import (
+    EntitySerializer,
+)
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework.reverse import reverse
+
+
+class EntityViewSet(viewsets.ModelViewSet):
```

Comment:
I think we want a way to get entities by `wikidata_id` and `name` as well as by their internal ID